cfn-resources/database-user/resource-role.yaml

```diff
@@ -15,6 +15,13 @@ Resources:
             Principal:
               Service: resources.cloudformation.amazonaws.com
             Action: sts:AssumeRole
+            Condition:
```

Comment:
why has this file changed?

Reply:
thanks, i guessed that, but this probably means that we didn't do it in previous PRs